common/djangoapps/student/urls.py

```diff
@@ -24,12 +23,6 @@
 
     # password reset in views (see below for password reset django views)
     url(r'^account/password$', views.password_change_request_handler, name='password_change_request'),
```

Comment:
1. Can you comment on the PR to better clarify why this shouldn't move with the rest.
2. We should update the comment above this line.  Either remove it if it no longer makes sense, or update it based on how you answer this question.
Thanks.